app/mailers/referee_mailer.rb

```diff
@@ -8,6 +8,7 @@ def reference_request_email(application_form, reference)
     view_mail(GENERIC_NOTIFY_TEMPLATE,
               to: reference.email_address,
               subject: t('reference_request.subject.initial', candidate_name: @candidate_name),
+              reference: "#{Rails.env}-reference_request-#{reference.id}",
```

Comment:
Double checked that `mail_notify`'s `view_mail` call does support this and [it does](https://github.com/dxw/mail-notify/blob/master/README.md#with-optional-notify-arguments) 👍

Reply:
This won't work - on all live environments the `Rails.env` is "production". You probably want `HostingEnvironment.environment_name` ("production", "staging", "sandbox", etc), which is what we want to filter on.